apps/camera/js/lib/camera/camera.js

```diff
@@ -709,6 +709,21 @@ Camera.prototype.capture = function(options) {
 };
 
 /**
+ * Pauses/resumes a video capture session.
+ *
+ * @public
+ */
+Camera.prototype.pauseCapture = function() {
+  if (!this.mozCamera || this.mode !== 'video') {
+    return false;
```

Comment:
Same thing here as the patch on master regarding the return value. If this isn't used, just do `return;`.